deploy/cr.yaml

```diff
@@ -146,7 +146,7 @@ spec:
 
   backup:
     enabled: true
-    image: percona/percona-xtrabackup:8.0.28
+    image: perconalab/percona-server-mysql-operator:main-backup
```

Comment:
hope we will have it in next pxb release
https://github.com/percona/percona-docker/pull/629